schemas/NutritionSurvey.json

```diff
@@ -0,0 +1,264 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Nutrition Survey",
+  "description": "Data collection and quality assurance tools for Nutrition Surveys on mobile devices",
+  "type": "object",
+  "properties": {
+    "doc_type": {
```

Comment:
not a blocker, but would it be possible to have a fixed value here? `doc_type: 'nutrition-survey' or 'nutritionSurvey' or whatever ... i do not know which convention we are using for naming dot types